topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+>        - {% icon param-file %} *"Second input file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>        - *"first input file"*: `AMP`
+>        - *"Second input file"*: `TM `
+>
+>
+{: .hands_on}
+
+![Summary Plot](../../images/SummaryPlot.png "Summary plot shows comparisoin between AMPs and TMPs")
+
+
+
+### Assessing feature space distribution 
+
+In this tool, we have used **PDAUG Fisher's Plot** that compare two peptide library based on the feature space using the Fisher test. 
+
+
+> ### {% icon hands_on %} Hands-on: Generating a Fisher's plot to assess peptide dataset
+> 
+> 1. {% tool [PDAUG Fisher's Plot](toolshed.g2.bx.psu.edu/repos/jay/pdaug_fishers_plot/pdaug_fishers_plot/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"First fasta file"*: `PDAUG TSVtoFASTA on data 1 - first (fasta)` (first output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - {% icon param-file %} *"Second fasta file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - *"Label for first population"*: `AMP`
+>    - *"Label for second population"*: `TM`
+>
+>
+{: .hands_on}
+
+![Fishers plot](../../images/FeatureSpace.png "TMPs peptides show amino acids with larger hydrophobic residues in compare to AMPs")
+
+The AMPs and TMPs in the feature space represented by their mean hydropathy and amino acid volume. Fisher's plot shows that the sequences with larger hydrophobic amino acids are more frequent in TMPs in comparison to AMPs.
+
+## Assessing the relation between peptide features by 3D scatter plot
+
+### Calculating Sequence Property-Based Descriptors 
+
+In this step, we will be utilizing the "PDAUG Sequence Property Based Descriptors" tool to calculate CTD (Composition Transition and Distribution) descriptor.
+
+
+> ### {% icon hands_on %} Hands-on: Calculating descriptors for the peptide dataset
+> 
+> 1. {% tool [PDAUG Sequence Property Based Descriptors](toolshed.g2.bx.psu.edu/repos/jay/pdaug_sequence_property_based_descriptors/pdaug_sequence_property_based_descriptors/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `PDAUG TSVtoFASTA on data 1 - first (fasta)` (first output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - *"Descriptor Type"*: `CTD`
+>
+> 1. {% tool [PDAUG Sequence Property Based Descriptors](toolshed.g2.bx.psu.edu/repos/jay/pdaug_sequence_property_based_descriptors/pdaug_sequence_property_based_descriptors/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input fasta file"*: `PDAUG TSVtoFASTA on data 1 - second (fasta)` (second output of **PDAUG TSVtoFASTA** {% icon tool %})
+>    - *"Descriptor Type"*: `CTD`
+>
+>
+{: .hands_on}
+
+
+
+### Adding Class labels 
+
+- **Adding class label in AMPs data**
+
+> ### {% icon hands_on %} Hands-on: Adding class labels to the data frames
+> 
```

Comment:
replace “data frame(s)” with the more common Galaxy names: “tabular data” or “tabular file”

Reply:
Replaced

Reply:
@foellmelanie I would like to thank you and highly appreciate your time and effort to review these tutorials. Please let me know if there is any suggestion, as your feedback is of utmost importance. Thank you.